app/src/main/java/org/fossasia/openevent/app/module/event/create/CreateEventPresenter.java

```diff
@@ -25,6 +25,9 @@
 
 public class CreateEventPresenter extends BasePresenter<ICreateEventView> implements ICreateEventPresenter {
 
+    public static String ERROR_DATE_END_AFTER_START_MESSAGE = "End time should be after start time";
+    public static String ERROR_DATE_WRONG_FORMAT_MESSAGE = "Please enter date in correct format";
+    public static String SUCCESS_MESSAGE = "Event Created Successfully";
```

Comment:
Don't store in contants as we'll be moving them to strings.xml in future. If you want to reference them in tests, just do it manually for now

Reply:
Okay!